### PR TITLE
Detect pool user in script

### PR DESCRIPTION
**Pull request by pakrym**
Now requires `WebAdministration` powershell module to get information.

@blowdart @shirhatti

### CODE REVIEW

**Comment by shirhatti** [comment]
The `WebAdministration` module is not available on Nano. Perhaps on Nano you want to use the [`IISAdministration` module](https://technet.microsoft.com/en-us/library/mt270166.aspx) instead.
```
if (($EditionId -eq "ServerStandardNano") -or
    ($EditionId -eq "ServerDataCenterNano") -or
    ($EditionId -eq "NanoServer") -or
    ($EditionId -eq "ServerTuva"))
{
  ...
}
```

File: Provision-AutoGenKeys.ps1 [hunk 1]
```diff
@@ -69,14 +69,46 @@ function Provision-AutoGenKeys {
...
+else
+{
+    Import-Module WebAdministration
+    $processModel = Get-ItemProperty -Path "IIS:\AppPools\$appPoolName" -Name "processModel"
+}
+
+$identityType = $processModel.identityType
+Write-Output "Pool process model: '$identityType'"
+
+Switch ($identityType)
```
Comment (shirhatti) [inline Provision-AutoGenKeys.ps1]: What about `"SpecificUser"`?

Comment (pakrym) [inline Provision-AutoGenKeys.ps1]: Done

**Comment by shirhatti** [comment]
Looks good to me :shipit:

**Pull request merged**